### DISCUSSION TITLE
Download Directory

### DISCUSSION THREAD

**Discussion by deadbeatdandylyon** [Avnsx/fansly#22] [General]
Hi, I'm new to this, and new to this software, I tried it works like a charm, however I was wondering if there is a way to change the directory in which the download saves. or is it by default your main C:?

**Comment by Avnsx** [comment]
> Hi, I'm new to this, and new to this software, I tried it works like a charm, however I was wondering if there is a way to change the directory in which the download saves. or is it by default your main C:?

Here try this fork of my repo, it has download directories changing functionality https://github.com/pawnstar81/fansly

**Reply by Avnsx** [reply]
@pawnstar81 if you want, you could open a pull request and I'll merge your changes

**Reply by deadbeatdandylyon** [reply]
I looked at the coding, does it ask in the CMD prompt window when and where you want to download the files to?

**Reply by Avnsx** [reply]
> I looked at the coding, does it ask in the CMD prompt window when and where you want to download the files to?

Oh my bad, I thought he also had added file directory changing, but he only added saved media seperation into subfolders.

Ok try this repo instead: https://github.com/romfetchr/fansly-dl
This guy added download directory changing together with alot of other stuff that I don't really like, which was the reason I didn't merge it

**Reply by deadbeatdandylyon** [reply]
Yeah I probably wouldn't be asking for help with this of if this was an option if my C: drive wasn't maxed out with data,

**Reply by Avnsx** [reply]
> Yeah I probably wouldn't be asking for help with this of if this was an option if my C: drive wasn't maxed out with data,

Well I would add all of the features and bug fixes myself but right now I have a huge ongoing private project, which leaves me no time to really maintain this repo too much. Right now I'm only willing to do anything, if my scraper is entirely not able to do it's main job of scraping due to fansly updates or whatever

**Reply by deadbeatdandylyon** [reply]
oh no i wasn't getting your case about anything, just explaining why i was wondering

**Reply by deadbeatdandylyon** [reply]
download_dir = E:\

so would this be the correct formula to specify the drive?

**Reply by Avnsx** [reply]
> download_dir = E:\
> 
> so would this be the correct formula to specify the drive?

Just open the location you want to save the files in, inside windows explorer and then you can click on the location hireachy bar on the top to copy paste it into fansly scrapers config file

**Reply by deadbeatdandylyon** [reply]
Ahh, I went the easy route and just copied and pasted the extracted zip folder of Fansly scraper into the E drive and its working that way

**Reply by deadbeatdandylyon** [reply]
Hey got a question. I just spent like the last hour or so downloading, and all but one file is working the rest say for photos the file is either unsupported or corrupted, and the videos just won't play and says to check the log file. is this because i copied the extracted fansly_scraper folder to a different drive instead of just doing the download_dir?

**Reply by Avnsx** [reply]
> Hey got a question. I just spent like the last hour or so downloading, and all but one file is working the rest say for photos the file is either unsupported or corrupted, and the videos just won't play and says to check the log file. is this because i copied the extracted fansly_scraper folder to a different drive instead of just doing the download_dir?

I don't know ¯\\_(ツ)_/¯ Sounds like a issue with the file types (.png, .mp4 etc.).

**Comment by Kobalt77** [comment]
Hi,

Does this function has been removed from the program??
As it was not here in the .ini file, I added the line to change the download directory
But it doesn't seem to work...
![notepad++_2023-01-15_01-27-25](https://user-images.githubusercontent.com/38300027/212503029-e46e9340-8e1f-4474-bce6-42a8ff4930b2.jpg)

**Reply by Avnsx** [reply]
The last version that is published as compliled executable is version 0.3.5.
But the download directory feature will be added in 0.3.6, coming with the next update.
That version is not published yet, it's still in development.
You can only use download directory atm, if you download and use the raw python source.
